examples/java/java-sin/README.md

```diff
@@ -0,0 +1,16 @@
+# Java `sin` function
+
+This example uses the [Java Client](https://github.com/wandb/client-ng-java) to plot a
```

Comment:
This should be moved to client-java

Reply:
Are you ask the whole readme file to be moved? The client library lives at client-ng-java which has a readme file.

client-java is the old implementation which uses py4j.

Reply:
yes

Reply:
And update the Readme to be accurate post release,  no "--pre"